vertical-pod-autoscaler/e2e/v1/recommender.go

```diff
@@ -145,6 +147,43 @@ var _ = RecommenderE2eDescribe("Checkpoints", func() {
 	})
 })
 
+var _ = RecommenderE2eDescribe("CronJob Recommendation", func() {
+	f := framework.NewDefaultFramework("vertical-pod-autoscaling")
+
+	ginkgo.It("recommendations for CronJob are served", func() {
+		cronJob := framework.NewTestCronJob("hamster-cronjob", "*/3 * * * *", batchv1beta1.AllowConcurrent,
+			[]string{"sleep", "10000000"}, 2, 2, 10, nil, nil)
```

Comment:
This already sets up the container that we later overwrite. Would it be possible to pass the correct command here?

Also, can you make it more explicit what the constants are? (the 2, 2, 10)